crates/primitives/src/constants/eip4844.rs

```diff
@@ -35,10 +38,10 @@ pub const VERSIONED_HASH_VERSION_KZG: u8 = 0x01;
 const TRUSTED_SETUP_RAW: &[u8] = include_bytes!("../../res/eip4844/trusted_setup.txt");
 
 /// KZG trusted setup
-pub static KZG_TRUSTED_SETUP: Lazy<Arc<KzgSettings>> = Lazy::new(|| {
-    Arc::new(
+pub static KZG_TRUSTED_SETUP: Lazy<Arc<Mutex<KzgSettings>>> = Lazy::new(|| {
+    Arc::new(Mutex::new(
```

Comment:
this will be problematic, because this will be used by tx validator.

we should keep this and rename to something like MAINNET trusted setup or something

and configure the `Arc<KzgSettings>` via NodeCommand, see EthTransactionBuilder for example which has a fn to set the KzgSettings

Reply:
> this will be problematic, because this will be used by tx validator.

Could you expand on this?

> we should keep this and rename to something like MAINNET trusted setup or something

So, instead of a `pub static KZG_TRUSTED_SETUP`, a `pub static MAINNET_TRUSTED_SETUP`? Should it require then that `mainnet` is passed as the chain for us to allow passing `--trusted-setup-file`?

Reply:
Ok, lmk if this is good now! @mattsse